package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oat-sa/prettier-config",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
**issue:** The version bump will be made by the release tool. It should not be done manually, unless you want to jump over a version...

Reply:
true! I just fixed it. Thanks!